inspirehep/modules/records/jsonschemas/records/elements/id.json

```diff
@@ -0,0 +1,155 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "anyOf": [
+        {
+            "properties": {
+                "type": {
+                    "enum": [
+                        "INSPIRE ID"
+                    ],
+                    "type": "string"
+                },
+                "value": {
+                    "pattern": "INSPIRE-\\d+",
```

Comment:
Yes, they have a fixed length of 8 digits padded with 0s. (we should normalize those who have a wrong padding).